src/org/rascalmpl/library/Prelude.java

```diff
@@ -2449,12 +2446,12 @@ public IFunction loadParser(IValue grammar, ISourceLocation savedLocation, IBool
 	
 	// REFLECT -- copy in {@link PreludeCompiled}
 	protected IConstructor makeConstructor(TypeStore store, Type returnType, String name, IValue ...args) {
-	    IValue value = values.constructor(store.lookupConstructor(returnType, name, TypeFactory.getInstance().tupleType(args)), args, new HashMap<String, IValue>());
-        Type type = value.getType();
-        if (type.isAbstractData()) {
-            return (IConstructor)value;
-        }
-        throw RuntimeExceptionFactory.implodeError("Calling of constructor " + name + " did not return a constructor");
+		IValue value = values.constructor(store.lookupConstructor(returnType, name, TypeFactory.getInstance().tupleType(args)), args, new HashMap<String, IValue>());
```

Comment:
there are quite some changes in this file with tabs & spaces replaced, lets try and reduce that.

In the future we're going to normalize all the tabs and spaces, but lets not polute the history of these lines by these commits.